### ISSUE TITLE
Integration Test for Bible -> HTML

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/door43.org#417]
## Story

As a *user* I don't want to see a 404 page on my project page so that I don't get scared or frustrated.

Task is to write a basic integration test to make sure that the follow repo types are converted without failing.

* [ ] Pick a tS upload
* [ ] Pick a USFM bundle

See #384 for a list of repos that are currently having problems.

## Notes

Setup tx-manager-lambda in Travis CI to initiate the integration tests `master`.

**Comment by phillip-hopper** [comment]
Now blocked by https://github.com/unfoldingWord-dev/door43.org/issues/441

**Comment by PhotoNomad0** [comment]
Preliminary Test Process:
- get latest commit ID for repo (for nowuse beautifulsoup to extract from commits page until API is finished)
- build dummy webhook json data
- launch client webhook: `ClientWebhook(**env_vars).process_webhook()`
- poll the build_log.json on the cdn_bucket until either errors or ended_at.

**Comment by PhotoNomad0** [comment]
Talking with @richmahn he suggests the best way to do an end to end integration test would be on dev.  On merge of tx-manager-lambda to master, dev would already have the latest code and we would be able to do the conversion end to end there after clearing out dev-cdn.door43.org/u/<user>/<repo>/<commit> and the preconvert repo.  That way we could run this as a unit test on Travis before deploying to production.

**Comment by PhotoNomad0** [comment]
Current conversion failures from list on https://github.com/unfoldingWord-dev/door43.org/issues/384:
- https://git.door43.org/lversaw/awa_act_text_reg.git  - Expected end of text (at char 24993), (line:292, col:121) backslash in text
- https://git.door43.org/Zipson/kfc_act_text_udb.git  - doesn't generate output file
- https://git.door43.org/vedhanthavijay/kpb_mat_text_udb.git  - preconvert leaves backslash at end of line

**Comment by PhotoNomad0** [comment]
@jag3773 
- have two repos that fail (above) due to an extra backslash in preprocessed files.  Do you want me to take a look into that?  Or should we open another issue for later?
- should we fork some of the test cases into tx-manager-test-data?  Just thinking they could be deleted renamed at any time.
- I have picked https://git.door43.org/tx-manager-test-data/en-obs-rc-0.2.git for the OBS conversion test.  Do you have a better option?

**Comment by jag3773** [comment]
This task is for scripture so OBS is not part of it.

Yes, use tx-manager-test .

Sounds like the two repos you found are perfect test cases, the conversion should not fail because of that, but it should be gracefully handling that case.
